utils/nft-utils/src/main.rs

```diff
@@ -0,0 +1,217 @@
+use std::thread;
+use std::time::Duration;
+
+use borsh::ser::BorshSerialize;
+use demo_stf::runtime::RuntimeCall;
+use sov_modules_api::default_context::DefaultContext;
+use sov_modules_api::default_signature::private_key::DefaultPrivateKey;
+use sov_modules_api::transaction::Transaction;
+use sov_modules_api::{Address, PrivateKey};
+use sov_nft_module::utils::get_collection_address;
+use sov_nft_module::{CallMessage, CollectionAddress, UserAddress};
+use sov_rollup_interface::mocks::MockDaSpec;
+use sov_sequencer::utils::SimpleClient;
+
+const COLLECTION_1: &str = "Sovereign Squirrel Syndicate";
+const COLLECTION_2: &str = "Celestial Dolphins";
+const COLLECTION_3: &str = "Risky Rhinos";
+
+const DUMMY_URL: &str = "http://foobar.storage";
+
+const PK1: [u8; 32] = [
+    199, 23, 116, 41, 227, 173, 69, 178, 7, 24, 164, 151, 88, 149, 52, 187, 102, 167, 163, 248, 38,
+    86, 207, 66, 87, 81, 56, 66, 211, 150, 208, 155,
+];
+const PK2: [u8; 32] = [
+    92, 136, 187, 3, 235, 27, 9, 215, 232, 93, 24, 78, 85, 255, 234, 60, 152, 21, 139, 246, 151,
+    129, 152, 227, 231, 204, 38, 84, 159, 129, 71, 143,
+];
+const PK3: [u8; 32] = [
+    233, 139, 68, 72, 169, 252, 229, 117, 72, 144, 47, 191, 13, 42, 32, 107, 190, 52, 102, 210,
+    161, 208, 245, 116, 93, 84, 37, 87, 171, 44, 30, 239,
+];
+
+fn get_collection_metadata_url(collection_address: &str) -> String {
+    format!("{}/collection/{}", DUMMY_URL, collection_address)
+}
+
+fn get_nft_metadata_url(collection_address: &str, nft_id: u64) -> String {
+    format!("{}/nft/{}/{}", DUMMY_URL, collection_address, nft_id)
+}
```

Comment:
This package mixes real functionality and mocks. Can we separate it cleanly into module with real helpers ("create collection", etc.), and then a binary with tge dummy data and main function? That way, someone could depend on the helpers and get a working implementation by just passing some actual data instead of mocks